src/lib/task/call.cc

```diff
@@ -186,6 +186,9 @@ int process_bam(task::Call::argument_type &arg) {
     Pedigree ped = io::parse_ped(arg.ped);
 
     // Open Reference
+    if(arg.fasta.empty()){
+    	throw std::runtime_error("Path to fasta file is empty.");
```

Comment:
Let's provide a better error message.

throw std::invalid_argument("Path to reference file must be specified with --fasta when processing bam/sam/cram files.")